### ISSUE TITLE
ParseBatch doesn't exist in jar file

### ISSUE THREAD

**Issue by tariqfcb** [thiagolocatelli/parse4j#45]
I cannot use ParseBatch in my application, when I try it I am getting : "ParseBatch cannot be resolved to a type"

I also checked the classes in the jar library and I cannot find the ParseBatch class under org.Parse4j

**Comment by nickolayrusev** [comment]
yes, they will be available in the next release when the maintainer include latest changes. may be the wrong thing is that they are included in the readme (my bad). Possible workaround is to download manually the library and include it in your project...

**Comment by JackP78** [comment]
Would also like to add that the ParseBatch that is currently in source control has a bug in it.  ParseObjects that are created using the ParseBatch do not have their ObjectId set which makes it impossible to later update them or associate them with other ParseObjects

**Comment by nickolayrusev** [comment]
thanks for your feedback. I made the batch functionality and when i have spare time will look into this .

**Comment by alexargo** [comment]
Just found this library and this got me too. FYI, temporarily you can use a specific commit if you add a new maven repository using jitpack.io and specify a commit:

Repository:

    <repositories>
        <repository>
            <id>jitpack.io</id>
            <url>https://jitpack.io</url>
        </repository>
    </repositories>

Dependency:

      <dependency>
          <groupId>com.github.thiagolocatelli</groupId>
          <artifactId>parse4j</artifactId>
          <version>036c44ca4eb167a260897ddd9e573dc63f72d796</version>
      </dependency>

@thiagolocatelli : Any idea when you plan on doing a new release?
